internal/services/redis/redis_firewall_rule_resource_test.go

```diff
@@ -99,7 +99,11 @@ func (t RedisFirewallRuleResource) Exists(ctx context.Context, clients *clients.
 func (RedisFirewallRuleResource) basic(data acceptance.TestData) string {
 	return fmt.Sprintf(`
 provider "azurerm" {
-  features {}
+  features {
+    resource_group {
+      prevent_deletion_if_contains_resources = false
```

Comment:
this would indicate something isn't being deleted correctly which would cause issues for real users - rather than adding this workaround, can we fix the issue here?